/**:
  ros__parameters:
    look_ahead_distance: 0.1
    speed: 0.1
    ignore_z_distance: false
    path_file: ""
